### PR TITLE
#160447609 Analytics for duration of all meetings per week

### PR DESCRIPTION
**Pull request by kzyGit**
#### What does this PR do?
- Adds Analytics for duration of all meetings per week

#### How should this be manually tested?
- Run the server
- Test query at `localhost:5000/mrm`
- Run `weeklyDurationsOfMeetings` query using parameters: `startDate`, `endDate` and `locationId`

#### What are the relevant pivotal tracker stories?
[#160447609](https://www.pivotaltracker.com/story/show/160447609)

#### Screenshots
screen shot 2018-10-02 at 09 04 27

### CODE REVIEW

File: fixtures/room/room_analytics_duration_fixtures.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                    "events": []
+                }
+            ]
+        }
+    }
+}
+
+get_weekly_meetings_total_duration_query = '''
+query {
+    weeklyDurationsOfMeetings(locationId:1, weekStart:"Jan 3 2018", weekEnd:"Jan 9 2021"){  # noqa: E501
```
Comment (3V0L) [inline fixtures/room/room_analytics_duration_fixtures.py]: I see here that the week End date is Jan 9 2021. Is this a typo?

File: helpers/calendar/analytics.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ class RoomStatistics(graphene.ObjectType):
     has_events = graphene.Boolean()
 
 
-class RoomDailyDurations(graphene.ObjectType):
+class RoomMeetingDurations(graphene.ObjectType):
```
Comment (sam-karis) [inline helpers/calendar/analytics.py]: Good work @kzyGit. I suggest you condense `RoomMeetingDurations` class to `RoomStatistics` by adding `total_duration` as an attribute.

Comment (kzyGit) [reply helpers/calendar/analytics.py]: Thanks for the feedback @sam-karis . I have resolved it already

File: fixtures/room/room_analytics_duration_fixtures.py (hunk 1 above)
Comment (kzyGit) [reply fixtures/room/room_analytics_duration_fixtures.py]: @3V0L . Thank you for identifying this. Solved already

**Pull request merged**